spec/system/comments_spec.rb

```diff
@@ -317,20 +402,20 @@
   end
 
   scenario "Reply to reply" do
-    create(:comment, commentable: debate, body: "Any estimates?")
+    create(:comment, commentable: resource, body: "Any estimates?")
 
-    login_as(create(:user))
-    visit debate_path(debate)
+    login_as(create(:user, :level_two))
```

Comment:
Would it be possible to add the reason why we add `:level_two` to the commit message?

Reply:
We are solved this question adding:

```
  let(:user) do
    if factory == :legislation_question
      create(:user, :level_two)
    else
      create(:user)
    end
  end
  ```
  
  and use `login_as(user)` in the test. This behaviour was to allow skip `require_verified_resident_for_commentable?` condition in comments/form_component.html.erb.